### PR TITLE
normalize storybook fixtures

### PR DESCRIPTION
**Pull request by radium-v**
# Pull Request

## 📖 Description

Normalizes all storybook stories by updating types, implementing a `renderComponent` helper function, and adding conventions for the optional, story-specific `storyContent` and `storyItems` args.

## 👩‍💻 Reviewer Notes

I moved any types and enum-style consts to the separate `*options.ts` file for each component, which helps isolate those simple values and types for both Storybook and Playwright usage. This will be more important on the Playwright side, since PW's TypeScript handling runs into errors when trying to load any component modules in its node source.

## 📑 Test Plan

All tests should pass as expected.

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## ⏭ Next Steps

This PR is split from the work in #6189.

### CODE REVIEW

File: packages/web-components/fast-foundation/src/listbox-option/stories/listbox-option.stories.ts [hunk 1]
```diff
@@ -1,79 +1,43 @@
...
-type ListboxOptionStoryArgs = Args & FASTListboxOption;
-type ListboxOptionStoryMeta = Meta<ListboxOptionStoryArgs>;
-
-const storyTemplate = html<ListboxOptionStoryArgs>`
+export const storyTemplate = html<StoryArgs<FASTListboxOption>>`
     <fast-option
-        :disabled="${x => x.disabled}"
+        ?disabled="${x => x.disabled}"
         ?selected="${x => x.selected}"
         :checked="${x => x.checked}"
```
Comment (radium-v) [inline packages/web-components/fast-foundation/src/listbox-option/stories/listbox-option.stories.ts:10]: The `checked` property should not be configurable here.

File: packages/web-components/fast-foundation/src/flipper/stories/flipper.stories.ts [hunk 2]
```diff
@@ -1,60 +1,28 @@
...
         ?disabled="${x => x.disabled}"
+        :hiddenFromAT="${x => x.hiddenFromAT}"
     ></fast-flipper>
 `;
 
 export default {
     title: "Flipper",
+    args: {
+        disabled: false,
+        hiddenFromAT: false,
```
Comment (radium-v) [inline packages/web-components/fast-foundation/src/flipper/stories/flipper.stories.ts:19]: `hiddenFromAT` should be true by default.

File: packages/web-components/fast-foundation/src/tooltip/stories/tooltip.stories.ts [hunk 3]
```diff
@@ -1,56 +1,50 @@
...
+    </fast-tooltip>
 `;
 
 export default {
     title: "Tooltip",
     args: {
-        content: "Here's a tooltip",
+        position: TooltipPosition.bottom,
+        storyContent: "Tooltip",
+        autoUpdateMode: AutoUpdateMode.anchor,
```
Comment (radium-v) [inline packages/web-components/fast-foundation/src/tooltip/stories/tooltip.stories.ts:26]:
```suggestion
        storyContent: "Tooltip",
```

File: packages/web-components/fast-foundation/.eslintrc.json [hunk 4]
```diff
@@ -1,6 +1,13 @@
 {
     "extends": ["@microsoft/eslint-config-fast-dna", "prettier"],
     "rules": {
+        "import/extensions": [
```
Comment (nicholasrice) [inline packages/web-components/fast-foundation/.eslintrc.json:4]: Is this to enforce extensions in TypeScript src files? If so I don't think it's necessary. ModuleResolution Node16 will throw compile errors if extensions don't exist.

File: change/@microsoft-fast-foundation-6f10b9c9-3154-4b85-a95c-f22a452e7d7a.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "normalize storybook stories",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "863023+radium-v@users.noreply.github.com",
+  "dependentChangeType": "patch"
```
Comment (nicholasrice) [inline change/@microsoft-fast-foundation-6f10b9c9-3154-4b85-a95c-f22a452e7d7a.json:6]:
```suggestion
  "dependentChangeType": "prerelease"
```

**Review by nicholasrice — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]
Lotta work here - thank you @radium-v!

**Review by EisenbergEffect — APPROVED** [review]
This is a massive accomplishment. Thanks for putting this together @radium-v. My review was pretty high-level so if there are errors in the details, I probably didn't catch them. However, from file to file things look consistent and an improvement from before. Very glad to see this coming together.

File: packages/web-components/fast-foundation/.eslintrc.json (hunk 4 above)
Comment (radium-v) [reply packages/web-components/fast-foundation/.eslintrc.json:4]: This was mistakenly added to this PR - it was needed during the playwright test file conversions.

**Pull request merged**